matroska_tags.xml

```diff
@@ -251,6 +251,16 @@ an age in other countries or a URI defining a logo).</description>
     <tag name="DATE_PURCHASED" class="Temporal Information" type="UTF-8">
       <description lang="en">Information on when the file was purchased; see also (#commercial) on purchase tags.</description>
     </tag>
+    <tag name="DATE_STARTED" class="Temporal Information" type="UTF-8">
+      <description lang="en">When the information of the parent tag starts being valid.
+      The information of the parent tag is only valid between this date and the `DATE_ENDED` date of the same level.
+      The `DATE_ENDED` is **OPTIONAL**, in which case the end date is unknown.</description>
```

Comment:
I am not a native English speaker, but is this sentence clear? `In which case` sounds strange to me here.

(Also below, at line 262.)

Reply:
Good catch! I also fixed some bogus wording.